src/main/java/gov/cms/madie/models/measure/TestCase.java

```diff
@@ -68,6 +68,9 @@ public class TestCase implements Serializable, Cloneable {
   private String validationStatus;
   private String validationTaskId;
 
+  // Lock information - transient field populated on retrieval, not persisted
+  @JsonIgnore private TestCaseLockInfo testCaseLock;
```

Comment:
Wouldn't we want to use `@Transient`? Pretty sure `@JsonIgnore` will prevent the lock info from being included in service responses.